tasks/fix-cat2.yml

```diff
@@ -1583,6 +1598,7 @@
       name: net.ipv6.conf.all.accept_source_route
       present: yes
       value: 0
+  ignore_errors: yes
```

Comment:
Same here. Use `ignorerrors` in the module not the task. (`syscstl` is special in that it has its own `ignoreerrors` parameter).